lib/console.js

```diff
@@ -125,6 +125,7 @@ Console.prototype.timeEnd = function timeEnd(label) {
   const ms = duration[0] * 1000 + duration[1] / 1e6;
   this.log('%s: %sms', label, ms.toFixed(3));
   this._times.delete(label);
+	return ms.toFixed(3);//return the elapsed time
```

Comment:
Another suggestion... do the `ms.toFixed(3)` conversion only once... e.g.

`const ms = (duration[0] * 1000 + duration[1] / 1e6).toFixed(3);`